### ISSUE TITLE
Absorb image-related submodules into `cryojax.ndimage`

### ISSUE THREAD

**Issue by michael-0brien** [mjo22/cryojax#485]
I think that the average `cryojax` user has to interact with too many submodules. One issue is that to create a volume, a user has to interact with `cryojax.io`, `cryojax.constants`, and even at times `cryojax.atom_util`. This would be addressed by #481.

Another issue is that functions for operating on images is scattered: `cryojax.coordinates` is for pixel/voxel coordinate systems, `cryojax.ndimage` is for utility functions that expect pixels/voxels as input, `cryojax.ndimage.transforms` / `cryojax.ndimage.operators` is for callables that accept images / coordinates as input (filters, masks, and parametrized functions). I would suggest we absorb all of these into the `cryojax.ndimage` namespace, keeping the other aliases for deprecation. This will be easier to use and document.

`cryojax.coordinates` used to be separate because it was not necessarily going to be for pixels/voxels, but with the creation of `cryojax.atom_util` any atom-based coordinate operations can go in there.
